stripe/res/layout/stripe_billing_address_layout.xml

```diff
@@ -12,25 +12,18 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content">
 
-            <com.google.android.material.textfield.TextInputLayout
+            <com.stripe.android.view.CountryAutoCompleteTextView
                 android:id="@+id/country_layout"
                 style="@style/StripePaymentSheetAutocompleteTextInputLayout"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
-                app:helperTextEnabled="false">
-
-                <AutoCompleteTextView
-                    android:id="@+id/country"
-                    style="@style/StripePaymentSheetTextInputEditText"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:inputType="none"
-                    android:paddingVertical="13dp"
-                    android:paddingHorizontal="12dp" />
-            </com.google.android.material.textfield.TextInputLayout>
+                app:helperTextEnabled="false"
+                app:countryAutoCompleteStyle="@style/StripeBillingAddressCountryCell"
+                app:countryItemLayout="@layout/stripe_country_dropdown_item"
+            />
```

Comment:
Should we just make these two separate views? It seems like we have a lot of extra logic now in `CountryAutoCompleteTextView`. Would it be easier to just create a new `CountryAutoCompleteTextInputLayout`? In fact, I think we could accomplish this with a vanilla `TextInputLayout` and a specific style.

Reply:
hmm I think the value of having them in a single view is to be able to reuse the country logics (`CountryAdapter`, `countryChangeCallback`, `CountryAutoCompleteTextViewValidator` etc) and just stylize them, agreed the name should be updated, will change that

Reply:
If we go with this approach I think "TextField" would be a good name.

Reply:
> hmm I think the value of having them in a single view is to be able to reuse the country logics

I think we can still do this. Can all the logic continue to live in `CountryAutoCompleteTextView` and then we have a light `TextInputLayout` wrapper around that?